### ISSUE TITLE
Strawberry stopped using Kvantum color scheme

### ISSUE THREAD

**Issue by McUberStein** [strawberrymusicplayer/strawberry#1029]
See the FAQ before opening an issue: https://wiki.strawberrymusicplayer.org/wiki/FAQ
Check the Changelog to see if the issue is already fixed: https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog
If it's already fixed, try the latest development build from: https://builds.strawberrymusicplayer.org/
For technical issues, questions and discussion please use the forum on https://forum.strawberrymusicplayer.org/
Any issues related to feature requests will be closed. See the README for more details.

**Describe the bug**
Strawberry does not use the system default color scheme set by qt5ct and Kvantum. Started when I uninstalled Strawberry 1.0.7 to upgrade to version 1.0.8, then the issue started, I downgraded to 1.0.7 and haven't been able to get it to work properly since.

**To Reproduce**
Uninstall current Strawberry install, install new version (1.0.8)

**Expected behavior**
Strawberry should use the system default Qt color scheme (Sweet Dark) instead of the default theme.

**Screenshots:**
Strawberry not using system theme despite option being set: [screenshot](https://imgur.com/a/GgFfzaS)
Output when launching Strawberry from the terminal: [screenshot](https://imgur.com/a/1uDIn7O)

**System Information:**
 - Operating system: Gentoo Linux
 - Strawberry Version: 1.0.7

**Additional context**
Around when the issue started, I have been (unsuccessfully) trying to compile Qt6 from git, which could be related. All other apps (Dolphin, Alacritty, Rhythmbox, Audacity) work with my Kvantum color scheme. Deleting my config file did not work.

**Comment by jonaski** [comment]
There are no changes related to Qt style or theme icons between 1.0.7 and 1.0.8, this issue is probably some settings or distro specific, perhaps Gentoo changed something.

**Comment by McUberStein** [comment]
I figured out what the problem was. I noticed that when compiling Strawberry, CMake automatically finds and uses Qt6 instead of Qt5. It just so happens that I tried compiling Qt6 a while ago, the compilation failed, but for some reason I decided to do `cmake --install` anyway (my mistake). So I deleted every Qt6 file I could find, told Portage to recompile Strawberry and during compilation noticed that it was now using Qt5. My Kvantum color scheme was back when I opened Strawberry. This whole issue was my fault. My bad.
